ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/set/location/AlterTableSetLocationAnalyzer.java

```diff
@@ -56,8 +56,7 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
     } catch (FileNotFoundException e) {
       // Only check host/port pair is valid, whether the file exist or not does not matter
     } catch (Exception e) {
-      throw new SemanticException("Cannot connect to namenode, please check if host/port pair for " + newLocation +
-          " is valid", e);
+      throw new SemanticException("Unable to get path:" + newLocation , e);
```

Comment:
Unable to get Path isn't clear either, FNF is ignored, that means path is of least importance, but the connection to FileSystem layer is.
```
Can not connect to the server, please check if host/port pair for  + newLocation +
          " is valid", e);```

might be a possibility as Namenode is an HDFS only concept